src/platform/packages/shared/kbn-esql-ast/src/parser/cst_to_ast_converter.ts

```diff
@@ -2072,7 +2062,9 @@ export class CstToAstConverter {
 
     try {
       for (const aggField of ctx.aggField_list()) {
-        const field = this.fromField(aggField.field());
+        if (aggField.getText() === '') continue;
+
+        const field = this.fromAggField(aggField);
```

Comment:
STATS use fromAggField instead of fromAggFields. I think it is the same for INLINESTATS.
I noticed this from tests on where that failed because the getPosition() function received the argument name '=" instead of 'where'.
cc @vadimkibana